eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/CDDL.hs

```diff
@@ -853,3 +853,11 @@ nonempty_set = binding $ \x ->
   "nonempty_set"
     =:= tag 258 (arr [1 <+ a x])
     / sarr [1 <+ a x]
+
+-- | An OSet is a Set that preserves the order of its elements.
+oset :: IsType0 t0 => t0 -> GRuleCall
+oset = binding $ \x -> "oset" =:= tag 258 (arr [0 <+ a x])
+
+-- | An NonEmpty OSet is a NonEmpty Set that preserves the order of its elements.
+nonempty_oset :: IsType0 t0 => t0 -> GRuleCall
+nonempty_oset = binding $ \x -> "nonempty_oset" =:= tag 258 (arr [1 <+ a x])
```

Comment:
Both oset and non-empty oset do not require the 258 tag, it is still optional. See the implementation:
1. https://github.com/IntersectMBO/cardano-ledger/blob/47f4f4ed7f3d5ee95c6e2dd73c5c0eb412c8449f/libs/cardano-data/src/Data/OSet/Strict.hs#L109
2. https://github.com/IntersectMBO/cardano-ledger/blob/47f4f4ed7f3d5ee95c6e2dd73c5c0eb412c8449f/libs/cardano-ledger-binary/src/Cardano/Ledger/Binary/Decoding/Decoder.hs#L1046

Reply:
I somehow thought that we could enforce the tag, but I was wrong in doing that for the existing conway era :+1: Thanks for catching the error.